packages/slate-react/src/utils/get-selection-from-dom.js

```diff
@@ -0,0 +1,77 @@
+import findRange from './find-range'
+
+export default function getSelectionFromDOM(window, editor, domSelection) {
```

Comment:
Can you rename this to `getDomSelection`?

Reply:
Wouldn't the rename be confusing? It seems to imply that the return value would be a DOM selection. I named it this way because most methods in Slate appear written from the point of view of Slate. So in this instance, we are getting the Slate (implied) selection from the DOM selection. I can see my name was not clear (since it confused you) but perhaps something like `getSlateSelectionFromDomSelection`? I'm not passionate about it though so if you are still happy with `getDomSelection` I will change.

Reply:
Very fair. Okay let's keep the current naming and I can figure out a rename in the future after it's in master.